### ISSUE TITLE
Computing Model Performance

### ISSUE THREAD

**Issue by chinmay5** [AlexeyAB/darknet#1170]
We trained a model on a custom dataset and computed its performance based on mAP. However, we needed a good performance on a subset of the labels in the dataset. So, we took the trained model on the entire dataset and simply created another dataset with a reduced number of classes and ran the model on it (Please let me know if the procedure is not clear). Now, when I tried getting the mAP of this fine-tuned model on the original dataset(just to see if a comparison), the mAP value is really low. However, the F1 score has improved. Is this an acceptable situation?

**Comment by AlexeyAB** [comment]
> So, we took the trained model on the entire dataset and simply created another dataset with a reduced number of classes and **ran the model on it** (Please let me know if the procedure is not clear). 

Do you mean you run Detection and Calculation mAP on dataset (with few classes) by using model that is trained on dataset (with more classes)? In this case some of APs will be 0 so you can caluclate true mAP manually - just get averge non-zero APs.

Or did you finetuned 1st model on the new dataset and get new 2nd model, that you use currently? mAP is more important, so may be better to use old model.
